docs/best_practices/tool_xml.rst

```diff
@@ -135,6 +135,9 @@ escaping (``&amp;``).
 
 `Wikipedia has more on CDATA <http://en.wikipedia.org/wiki/CDATA>`__
 
+All Cheetah variables for text parameters, input and output files must be
+single-quoted, e.g. ``'${var_name}'``.
```

Comment:
Am I right in assuming dollar, single-quote, and double quote are by default sanitized?

I suppose that being cautious is wise here. However, this tiny change will mean updating loads of existing tools and documentation to follow this improved best practice.

(Might anyone have used double-quotes and allowed a single quote in the variable? e.g. as an apostrophe in a text field)